tests/test_playbooks/architecture.yml

```diff
@@ -61,10 +84,10 @@
   tasks:
   - include: tasks/operating_system.yml
     vars:
-      operating_system_name: "TestOS1"
-      operating_system_state: absent
-  - include: tasks/operating_system.yml
-    vars:
-      operating_system_name: "TestOS2"
+      operating_system_name: "{{ item }}"
       operating_system_state: absent
+    loop:
+      - "TestOS1"
+      - "TestOS1 SP1"
```

Comment:
I thought the name cannot contain spaces?

Reply:
I guess you are right. We can not be sure without attempting to rerecord.